R/colby_constructors.R

```diff
@@ -1834,9 +1857,76 @@ manual_cols <- function(..., .lst = list(...)) {
     label = names(.lst)
   ))
   ctree <- splitvec_to_coltree(data.frame(), splvec = splvec, pos = TreePos())
-  InstantiatedColumnInfo(treelyt = ctree)
+  ret <- InstantiatedColumnInfo(treelyt = ctree)
+  rm_all_colcounts(ret)
```

Comment:
Because counts are now calculated during the coltree creation process, which happens even in the case of manually constructed cols, but historically, the column counts for manually constructed columns have defaulted to NA, and in fact there is a test that implicitly enforces that they are missing.